smart-cards/src/main/java/gov/va/api/health/smartcards/WebExceptionHandler.java

```diff
@@ -174,7 +183,8 @@ OperationOutcome handleBadRequest(Exception e, HttpServletRequest request) {
   @ExceptionHandler(HttpClientErrorException.Forbidden.class)
   @ResponseStatus(HttpStatus.FORBIDDEN)
   OperationOutcome handleForbidden(Exception e, HttpServletRequest request) {
-    return responseFor("forbidden", e, request, emptyList(), true);
+    Optional<OperationOutcome> maybeOperationOutcome = operationOutcomeFromClientResponse(e);
+    return maybeOperationOutcome.orElse(responseFor("forbidden", e, request, emptyList(), true));
```

Comment:
I'm concerned we may not remember to do this check everywhere `HttpClientErrorException` pops up. Can we instead do this at the beginning of the existing `asOperationOutcome` method?